docs/Containerfile.5.md

```diff
@@ -216,7 +216,7 @@ The secret needs to be passed to the build using the --secret flag. The final im
   If you use the shell form of the **CMD**, the `<command>` executes in `/bin/sh -c`:
 
   Note that the exec form is parsed as a JSON array, which means that you must
-  use double-quotes (") around words not single-quotes (').
+  use double-quotes (") around words, not single-quotes (').
```

Comment:
Just noting that https://github.com/containers/buildah/pull/3746 and https://github.com/containers/common/pull/905 are moving this file from Buildah to c/common.  We should ensure these changes move too.  @Foxboron FYI

Reply:
I can produce a very ad-hoc patch and apply on top of my branch so @eriksjolund get author credit there :)